server/graphql/v2/interface/Collection.ts

```diff
@@ -54,4 +54,12 @@ export interface CollectionReturnType {
   offset: number;
 }
 
+export interface TransactionsCollectionReturnType {
```

Comment:
Extend `CollectionReturnType` with `kinds`.

Reply:
@kewitz What would be the syntax?